phases/phase-1/challenges/accept-referral.md

```diff
@@ -0,0 +1,28 @@
+# Accept a referral
+:::tip Note  
+This challenge is only available top users who have been referred from another user via the [Referral challenge](refer.md) and who have already created a post via the [Post challenge](create-post.md). 
+
+Upon completing this challenge, you will be rewarded with **50 Desmos Tokens**.   
```

Comment:
Upon completing this challenge, you will be rewarded with **50 Desmos Tokens**.  

->

Upon completion of this challenge, you will get **50 Desmos Tokens**. Each user (verified with an account on GitHub) can only be rewarded once for this challenge.